app/views/articles/_full_comment_area.html.erb

```diff
@@ -3,7 +3,11 @@
     <% if @article.show_comments %>
       <header class="relative flex justify-between items-center">
         <h3 class="fs-2xl fw-bold">Discussion</h3>
-        <div id="comment-subscription"></div>
+        <div id="comment-subscription">
```

Comment:
This allows Preact to VDOM diff correctly and replace the SSR button @ludwiczakpawel .